internal/serve/httphandler/tss_handler.go

```diff
@@ -0,0 +1,118 @@
+package httphandler
+
+import (
+	"net/http"
+
+	"github.com/stellar/go/support/log"
+	"github.com/stellar/go/support/render/httpjson"
+	"github.com/stellar/wallet-backend/internal/apptracker"
+	"github.com/stellar/wallet-backend/internal/serve/httperror"
+	"github.com/stellar/wallet-backend/internal/tss"
+	"github.com/stellar/wallet-backend/internal/tss/router"
+	"github.com/stellar/wallet-backend/internal/tss/store"
+	"github.com/stellar/wallet-backend/internal/tss/utils"
+)
+
+type TSSHandler struct {
+	Router            router.Router
+	Store             store.Store
+	AppTracker        apptracker.AppTracker
+	NetworkPassphrase string
+}
+
+type Transaction struct {
+	Operations []string `json:"operations"`
```

Comment:
I'd recommend we add `validate` flag here too.

Reply:
Done